src/main/scala/mesosphere/marathon/api/akkahttp/v2/AppsController.scala

```diff
@@ -222,6 +259,14 @@ class AppsController(
 
   private def getVersion(appId: PathId, version: Timestamp)(implicit identity: Identity): Route = ???
 
+  private def completeWithDeploymentForApp(appId: PathId, plan: DeploymentPlan) =
```

Comment:
Could you add a comment so refactoring would be easier? Is this used somewhere else?

Reply:
1. sure
2. yes, it is, but not in this PR